features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/assertj/ViewEventAssert.kt

```diff
@@ -587,6 +587,16 @@ internal class ViewEventAssert(actual: ViewEvent) :
         return this
     }
 
+    fun hasSlownessInfo(
+        slowFrames: List<ViewEvent.SlowFrame>,
+        slowFrameRate: Double,
+        freezeRate: Double
+    ): ViewEventAssert = apply {
+        assertThat(actual.view.slowFrames).isEqualTo(slowFrames)
+        assertThat(actual.view.slowFramesRate).isEqualTo(slowFrameRate)
+        assertThat(actual.view.freezeRate).isEqualTo(freezeRate)
```

Comment:
it is better also to give a custom message using `overridingErrorMessage` call like in the custom assertions below and above